pyspecdata/load_files/__init__.py

```diff
@@ -221,6 +221,19 @@ def look_inside(inp_directory):
     return retval
 
 
+def register_proc_lookup(newdict):
+    """this updates the dictionary at
+    pyspecdata.load_files.postproc_lookup
+    this is equivalent to passing newdict to the lookup
+    keyword argument of `find_file`.
+
+    Remember that you only have to pass the keyword argument or call this
+    function once per script, and the effects are persistent!
+    """
+    postproc_lookup.update(newdict)
+    return
+
+
```

Comment:
I think this is doing what you explain `[here](https://jmfrancklab.slack.com/archives/C06LNFNP2HM/p1737499687080149?thread_ts=1737467514.307219&cid=C06LNFNP2HM)`  but you made it a function so we can call whenever